nerfstudio/data/dataparsers/minimal_dataparser.py

```diff
@@ -0,0 +1,94 @@
+# Copyright 2022 The Nerfstudio Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+""" Data parser for pre-prepared datasets for all cameras, with no additional processing needed"""
```

Comment:
We should include some documentation on the expected/optional fields in the npz file.

Reply:
added a brief doc about this- wdyt?

Reply:
Thats probably fine for now. At some point we should probably write out the schema, ie - https://docs.nerf.studio/en/latest/quickstart/data_conventions.html#dataset-format
For example, this requires a "scene_box" which is not documented anywhere.